analysisservice/src/main/java/nu/yona/server/analysis/service/AnalysisEngineService.java

```diff
@@ -41,13 +40,40 @@
 @Service
 public class AnalysisEngineService
 {
+	private class UserAnonymizedEntityHolder
```

Comment:
Why is this an improvement? The `UserAnonymizedService` already caches the result.